adcm_aio_client/core/objects/cm.py

```diff
@@ -445,6 +451,14 @@ async def _get_hosts(self: Self, host: Host | Iterable[Host] | Filter) -> Iterab
         return hosts
 
 
+def default_exit_condition(job: "Job") -> bool:
+    return job.get_status() in DEFAULT_JOB_TERMINAL_STATUSES
+
+
+def until_timeout_reached(timeout: int | None) -> datetime:
+    return datetime.max if timeout is None else (datetime.now() + timedelta(seconds=timeout))  # noqa: DTZ005
```

Comment:
non-descriptive name, if it's aim is to determine timeout time, it should clearly say so

and it's usage is incorrect: it'll be infinite loop always

I suggest simply inlining this code in `wait` before the while loop